### DISCUSSION TITLE
Jobs stuck in ENQUEUD state when reducing org.jobrunr.backgroud-job-server.poll-interval-in-seconds

### DISCUSSION THREAD

**Discussion by igorg-il** [jobrunr/jobrunr#710] [Q&A]
Hello
We are using Jobrunr version 4.0.5. All worked fine until we decided to add org.jobrunr.backgroud-job-server.poll-interval-in-seconds 
We don't run many jobs in parallel (usually only single job) so we wanted to make job start ASAP.

When setting poll-interval-in-seconds is 5 (which is minimal allowed value) jobs always stuck in Enqueued state.
For other values we tried (e.g. 7 and 10) it sometimes stuck and sometimes start processing.
For higher values looks like it works as expected.

Do u have any idea what can be the problem?
thank you

**Comment by rdehuyss** [comment]
There is a big popup on the dashboard mentioning that there is a **new JobRunr version** available. It's there for a reason: bugfixes and performance improvements.

I'm afraid I cannot help you if you don't upgrade. JobRunr Pro offers realtime job processing.

**Reply by igorg-il** [reply]
Upgraded to 6.1.1. Same problem happens. Any ideas please?
            <dependency>
                <groupId>org.jobrunr</groupId>
                <artifactId>jobrunr-spring-boot-2-starter</artifactId>
                <version>6.1.1</version>
            </dependency>

**Reply by rdehuyss** [reply]
No, a lot of people are running JobRunr like you do... If you can reproduce this on a small GitHub repo, I can have a look.

Otherwise, feel free to reach out to take a JobRunr Pro version. It comes with instant job processing and support.
